### DISCUSSION TITLE
Can't test with the different chrome version (Compatibility testing)

### DISCUSSION THREAD

**Discussion by yushun32** [seleniumbase/SeleniumBase#3168] [Q&A] [answered]
I would like to test with an older Chrome version; however, it tests with the latest version. 

I used the command "sbase get chromedriver 109", but it should be testing with Chrome version 109, not the latest chrome version.

**Comment by mdmintz** [comment, marked as answer]
SeleniumBase doesn't download the actual browsers for you. Just the drivers. If you want to test on a specific browser version, you'll need to download and install that separately.

**Reply by yushun32** [reply]
Thankyou. But is there any solution that can make it faster, or do I have to uninstall the current version and download an older version?

**Reply by mdmintz** [reply]
If you have a way of downgrading the browser version, then that's the way to do it. I just test on the latest version of browsers because downgrading isn't as easy.
